includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -955,16 +963,16 @@ public function maybe_render_subscription_payment_method( $payment_method_to_dis
 
 		// If we couldn't find a Stripe customer linked to the account, fallback to the order meta data.
 		if ( ( ! $stripe_customer_id || ! is_string( $stripe_customer_id ) ) && false !== $subscription->get_parent() ) {
-			$parent_order       = wc_get_order( $subscription->get_parent_id() );
+			$parent_order       = WC_Stripe_Order::get_by_id( $subscription->get_parent_id() );
 			$stripe_customer_id = $parent_order->get_meta( '_stripe_customer_id', true );
-			$stripe_source_id   = $parent_order->get_meta( '_stripe_source_id', true );
+			$stripe_source_id   = $parent_order->get_source_id();
```

Comment:
Optional: In the function definition for `WC_Stripe_Order::get_source_id()` what do you think about always passing `true` for the second param of `WC_Order::get_meta()`? This way, we are always sure to get a single string.

Reply:
I was referring to this: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/8b80ffb3caf4588cef777fd06ebcec9ba3b9ec5f/includes/class-wc-stripe-order.php#L814

Reply:
The default value for the second param is already `true`, so I removed it here. Do you see any value in adding it anyway?